packages/@dcl/react-ecs/src/components/utils.ts

```diff
@@ -15,3 +22,51 @@ export function parseProps(props: EntityPropTypes) {
     ...uiBackgroundProps
   }
 }
+
+/**
+ * @internal
+ */
+function getScaleAndUnit(scaleUnit: ScaleUnit): [number, ScaleUnits] {
+  if (typeof scaleUnit === 'number') {
+    return [scaleUnit, 'vw']
+  }
+
+  // the following regex would match any valid CSS unit (i.e: 7rem || 7% || 7vw)
+  const [_, value, unit] = scaleUnit.match(/(-?[\d.]+)([a-z%]*)/) as [string, number, ScaleUnits]
```

Comment:
maybe we should add both cases here instead of a regex ?
like 
```tsx
if (scaleUnity.endsWith('vw')) return [scaleUnity, 'vw']
if (scaleUnity.endsWith('vh')) return [scaleUnity, 'vw']
return [undefined, YGUnit.UNDEFINED]
```

This way if the user pass something that's not valid then we fallback to undefined. (A lot of users use any's) https://github.com/decentraland/js-sdk-toolchain/blob/main/test/react-ecs/transform.spec.tsx#L46